cv_eval.sh

```diff
@@ -0,0 +1,4 @@
+python src/cv_result_eval.py \
```

Comment:
Could you get the averaged score from 5-CV using this script?

Reply:
This script will get the average score of each combination of hyperparameters and will display the best hyperparameters. Do you want me to get the averaged score of all the results?